src/accelerate/utils/fsdp_utils.py

```diff
@@ -617,6 +617,7 @@ def fsdp2_prepare_model(accelerator, model: torch.nn.Module) -> torch.nn.Module:
         "offload_policy": fsdp2_plugin.cpu_offload,
         # `fully_shard` doesn't accept `None` in case of `MixedPrecisionPolicy`
         "mp_policy": fsdp2_plugin.mixed_precision_policy or MixedPrecisionPolicy(),
+        "device_mesh": accelerator.device_mesh[accelerator.parallelism_config.dp_dim_names]
```

Comment:
```suggestion
        "mesh": accelerator.device_mesh[accelerator.parallelism_config.dp_dim_names]
```
https://docs.pytorch.org/docs/stable/distributed.fsdp.fully_shard.html#torch.distributed.fsdp.fully_shard

Reply:
Good catch